bin/install

```diff
@@ -58,19 +58,16 @@ omf_install() {
     die "Could not clone the repository → ${OMF_PATH}:${OMF_REPO_BRANCH}"
   fi
 
-  pushd ${OMF_PATH} >/dev/null 2>&1
-
   local git_rev=$(git rev-parse HEAD) >/dev/null 2>&1
   local git_upstream=$(git config remote.upstream.url)
 
   if [ -z "${git_upstream}" ]; then
-    git remote add upstream ${git_uri}
+    git -C ${OMF_PATH} remote add upstream ${git_uri}
```

Comment:
Safer to use --git-dir? I am not confident old osx  versions have a new git version.